desktop/apps/artwork/components/auction/view.coffee

```diff
@@ -23,7 +26,9 @@ module.exports = class ArtworkAuctionView extends Backbone.View
     'change .js-artwork-auction-max-bid': 'setMaxBid'
 
   initialize: ({ @data }) -> #
-    setInterval before(10, @updateBidLabel), 1000
+    # Defer until next tick. This really just gets the sequence right for
+    # testing to work. This doesn't feel like a great justification, though.
+    setTimeout (=> @updateBidLabel LOT_STANDING_MAX_POLLS), 0
```

Comment:
This should get you what you need: https://github.com/artsy/force/blob/master/desktop/apps/artwork/client/test/index.coffee#L18 -- `benv`'s `expose` function.